### PR TITLE
Rename CONTRIBUTION.md to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ariyonaty**
## Description

In the effort of improving the state of the repo, this minor fix renames the contribution file from `CONTRIBUTION.md` to `CONTRIBUTING.md`, aligning with [GitHub's Default Health File Convention](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types). This also aligns with the majorly standardized convention employed by the majority of open-source repos.

## Type of change

- [x] Documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have maintained a clean commit history by using the necessary Git commands
- [x] I have checked that my code does not cause any merge conflicts

### CODE REVIEW

**Comment by Wahid7852** [comment]
Seriously...? This?
Where is the issue pointing out to this PR?

**Comment by ariyonaty** [comment]
@Wahid7852 - There is no issue linked here, nor am I doing this for any "points", "levels", or GSSoC events. 
Simply as I was poking around Hacktoberfest repos, noticed that the Github contribution guidelines section was not showing up - a result of the filename not following the convention laid out by Github and realized I can make the quick fix so that future viewers can have the more standardized experience.

**Review by techy4shri — COMMENTED** [review]
While I appreciate you including a link to GitHub's Default Health File Convention, know that this repository is currently under GirlScript Summer of Code-Extended also, apart from hacktober-fest, which means that we follow a set of rules for accepting pull requests. Even as a simple PR  in case this repository wasn't part of the program, you do not simply raise a PR unless stated by the maintainer.
You raise an issue first stating the problem and then raise a PR if you're assigned to that issue.

And seriously? Contributing.md?? That is not even a good name change....
Long story short: This PR is being rejected for not following the rules of the repo.
